init.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE txts (
```

Comment:
Table names should always have a meaningful and correct name.
Replace with texts

Reply:
@youcef-s

Reply:
done